### DISCUSSION TITLE
Add gravity to header?

### DISCUSSION THREAD

**Discussion by uSlackr** [thorrak/fermentrack#563] [Q&A] [answered]
Is there a way to add a box for gravity on the Temp controller dashboard? I'd rather see it than the Log Interval setting, for instance

Thanks

![image](https://user-images.githubusercontent.com/25751399/103246732-84321800-4932-11eb-8e98-f8ec6485daed.png)

**Comment by vanosg** [comment]
+1 ... If I had unlimited cosmic power, I'd move the profile block to where the configure device block currently is, and insert the gravity where the profile block currently is.

**Comment by thorrak** [comment, marked as answer]
Makes sense to me. Raised as issue #567
